products/knownprojects/sql/dcp_application.sql

```diff
@@ -68,24 +34,24 @@ zap_translated AS (
             WHEN dcp_projectphase::numeric = 717170004 THEN 'Public Completed'
             WHEN dcp_projectphase::numeric = 717170005 THEN 'Initiation'
         END) AS dcp_projectphase,
-        (CASE
-            WHEN dcp_projectcompleted IS NULL THEN NULL
-            ELSE TO_CHAR(dcp_projectcompleted::timestamp, 'YYYY/MM/DD')
-        END) AS dcp_projectcompleted,
         (CASE
             WHEN
                 COALESCE(
-                    dcp_certifiedreferred, dcp_dcptargetcertificationdate
+                    certified_referred, dcp_dcptargetcertificationdate
                 ) IS NULL
                 THEN NULL
             ELSE
                 TO_CHAR(
                     COALESCE(
-                        dcp_certifiedreferred, dcp_dcptargetcertificationdate
+                        certified_referred, dcp_dcptargetcertificationdate
                     )::timestamp,
                     'YYYY/MM/DD'
                 )
         END) AS dcp_certifiedreferred,
+        (CASE
+            WHEN completed_date IS NULL THEN NULL
+            ELSE TO_CHAR(completed_date::timestamp, 'YYYY/MM/DD')
+        END) AS completed_date,
```

Comment:
Nit, but you don't need the case (for this or the similar clause above)

`select to_char(null::timestamp, 'YYYY/MM/DD')` will return `NULL`

Reply:
word. I didn't add this though, it was on an earlier line and probably only moved via sqlfluff

Reply:
Wait what the heck really? Why is this computed column reordered differently than others? Bizarre

Reply:
hmmmm I was gonna say maybe it sorts alphabetically based on the column used, but that doesn't seem to be it

maybe I moved it and sqlfluff doesn't care since all the CASE statements are still grouped together?